File: Java/PrenotazioneCampoTennis/src/Socio.java
import java.util.*;

public class Socio 
{
	private String nomeCognome;
	private int codice, eta, categoria;
	protected List<Prenotazione> prenotazioni;
	
	public Socio(int c, String n, int e, int categoria)
	{
		codice = c;
		nomeCognome = n;
		eta = e;
		this.categoria = categoria;
		prenotazioni = new LinkedList<Prenotazione>();
	}
	
	public void addPrenotazione(Prenotazione p)
	{
		prenotazioni.add(p);
	}
	
	public String toString()
	{
		return codice + "\t" + nomeCognome + "\t" + eta + "\t" + categoria + "\t" + prenotazioni;
	}
}

File: SecondoAnno/Programmazione_II/Java/ESAMI/Esame24Gennaio2020/Python/Socio.py
class Socio:

    def __init__(self, codice, nome, eta, categoria) -> None:
        self.codice = codice
        self.nome = nome
        self.eta = eta
        self.categoria = categoria
        self.prenotazioni = []

    def addPrenotazione(self, pren):
        self.prenotazioni.append(pren)

    def __str__(self) -> str:
        return str(self.codice) + "\t" + self.nome + "\t" + str(self.eta) + "\t" + str(self.categoria) + "\t" + str([str(pren for pren in self.prenotazioni)])